packages/myst-frontmatter/src/contributors/validators.ts

```diff
@@ -127,16 +131,34 @@ export function validateName(input: any, opts: ValidationOptions) {
 /**
  * Validate Contributor object against the schema
  */
-export function validateContributor(input: any, stash: ReferenceStash, opts: ValidationOptions) {
+export function validateContributor(
+  input: any,
+  stash: ReferenceStash,
+  opts: ValidationOptions,
+): Contributor | undefined {
+  const inputAff = validateObjectKeys(
+    input,
+    { optional: AFFILIATION_KEYS, alias: AFFILIATION_ALIASES },
+    {
+      ...opts,
+      suppressErrors: true,
+      suppressWarnings: true,
+    },
+  );
+  if (inputAff?.collaboration === true) {
+    return validateAffiliation(input, opts);
+  }
   if (typeof input === 'string') {
     input = stashPlaceholder(input);
   }
-  const value = validateObjectKeys(
-    input,
-    { optional: CONTRIBUTOR_KEYS, alias: CONTRIBUTOR_ALIASES },
-    opts,
-  );
+  const value = validateObjectKeys(input, { optional: PERSON_KEYS, alias: PERSON_ALIASES }, opts);
   if (value === undefined) return undefined;
+  if (inputAff && Object.keys(inputAff).length > Object.keys(value).length) {
```

Comment:
This is a nice little check that says "Your input looks more like an affiliation than a person."